primefaces-themes/theme-base/_components.scss

```diff
@@ -3,6 +3,7 @@
 @import './_mixins';
 @import './_common';
 @import './_colors';
+@import './_keyframes';
```

Comment:
I am wondering if we should call keyframes as `animations`?  I am also ok leaving it keyframes.